src/bundle/indexing/warc-indexer.sh

```diff
@@ -160,6 +162,13 @@ check_parameters() {
     fi
 }
 
+# Either wget or curl must be present
+check_tools() {
+    if [[ -z $(which curl) && -z $(which wget) ]]; then
+        >&2 echo "Warning: Neither 'curl' nor 'wget' is present. ping and commit will be disabled"
```

Comment:
This warning is never shown on mac

Reply:
It's not enough to write the methods? We have to call them too?